app/src/features/apiClient/screens/apiClient/components/views/components/Collection/components/CollectionRunnerView/components/RunResultView/RunResultContainer/RunResultContainer.tsx

```
@@ -0,0 +1,394 @@
+        {requestNameDetails}
+        {responseDetails}
+      </div>
+
+      {testResults.length === 0 ? (
+        <div className="no-test-found-message">
+          <i>No test found</i>
+        </div>
+      ) : (
+        <div className="results-list">
+          {testResults.map((test) => {
+            return <TestResultItem testResult={test} />;
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)